sdk/daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/ledger/BlindingTransaction.scala

```diff
@@ -15,13 +15,14 @@ import com.daml.nameof.NameOf
 object BlindingTransaction {
 
   private object BlindState {
-    val Empty = BlindState(Relation.empty, Relation.empty)
+    val Empty = BlindState(Relation.empty, Relation.empty, Relation.empty)
   }
 
   /** State to use while computing blindingInfo. */
   private final case class BlindState(
       disclosures: Relation[NodeId, Party],
       divulgences: Relation[ContractId, Party],
+      visibility: Relation[ContractId, Party],
```

Comment:
Please add a comment, too, because `visibility` can mean many things to different people.

Reply:
I've renamed `contractVisibility`